nidmviewer/convert.py

```diff
@@ -24,7 +25,10 @@ def parse_coordinates(coordinates):
     count=0
     coordinate_df = pandas.DataFrame(columns=["x","y","z"])
     for coordinate in coordinates:
-        coordinate_df.loc[count] = [x.strip() for x in coordinate.strip("]").strip("[").split(",")]
+        if not isinstance(coordinate, float64) or not isnan(coordinate):
```

Comment:
and parsed as I had originally, it comes out cleanly:

![image](https://cloud.githubusercontent.com/assets/814322/15619402/ee4f3640-2408-11e6-9575-843efbdef3f8.png)